database/data.js

```diff
@@ -0,0 +1,178 @@
+    reviewer_name : faker.internet.userName(),
+    reviewer_city : faker.address.city(),
+    no_of_reviews : faker.random.number(),
+    reviewer_picture : faker.image.avatar(),
+    reviewer_VIP : (faker.random.boolean()).toString()
+  };
+
+  return reviewer_info;
+};
+
+// console.log(data_reviewer_info());
+
+
+var data_reviews = function(id0, id1){
+  var review = {
+    restaurant_id : id0,
+    review_id : id1,
+    reviews : faker.lorem.sentences(7)
+  };
+  return review;
+};
+
+// console.log(data_reviews(1));
+
+
+var dataWriteToDisk = function(fileName, callback){
+
+  var dataToStore = [];
+  var noOfRestaurants =  100;
+
+  if (fileName === 'reviews') {
+    // get 10 reviews per restaurant
+    for ( var i = 0; i < 10 ; i++){
+      for (var j = 0; j <= noOfRestaurants ; j++) {
+        dataToStore.push(callback(j+1, i+1));
+      }
+    }
+  } else {
+      for (var i = 1; i <= noOfRestaurants ; i++) {
+        dataToStore.push(callback(i));
+      }
+  }
+  fs.writeFile( __dirname + `/data/${fileName}.json`,
+         JSON.stringify(dataToStore), function(err){
+    if (err) {
+      throw err;
+    } else {
+      console.log(`SUCCESS : File ${fileName} created`);
+    }
+  })
+
+};
+
+dataWriteToDisk('ratings_ambience', data_ratings_ambience)
+dataWriteToDisk('reviewer_info', data_reviewer_info)
+dataWriteToDisk('reviews', data_reviews);
+
+
+
+// fs.readFile( __dirname + '/restaurants_id_name.csv', (err, data) => {
+//   if (err) {
+//     console.error(err)
+//     return
+//     }
+//   console.log(data)
+//   })
+
+// const results = [];
+
+// // fs.createReadStream(__dirname + '/restaurants_id_name.csv')
+// fs.createReadStream(__dirname + '/reviewer_info.csv')
+// .pipe(csv())
+// .on('data', (data) =>
+//     results.push(data))
+//     // console.log(data.id)
+//     // console.log(data.name)
+// .on('end', () => {
+//   console.log(results);
+//   // console.log(results[0].id);
+//   // console.log(results[0].name);
+
+// });
+
+// reviewer_info.json
+// reviews.json
+// ratings_ambience.json
+
+var dataReadFromDisk = function(fileName){
+  fs.readFile( __dirname + `/data/${fileName}.json`, (err, data) => {
+      if (err) {
+        console.error(' ERROR : Cannot read file');
+      }
+      var  student = JSON.parse(data);
+      // console.log(student)
+      console.log(student[0]);
+      // return JSON.parse(data);
+      return student[0];
+      // console.log(student[1]);
+
+  })
+};
+
+// dataReadFromDisk('reviewer_info');
+// reviewer_info.json
+// reviews.json
+// ratings_ambience.json
+
+module.exports = dataReadFromDisk;
+
+
+
+
+
+// // table - ratings_ambience
+// id
+// restaurant_id
+// reviews - faker.lorem.paragraph()
+
+
+// table - ratings_ambience
+// id
+// restaurant_name
+// restaurant_rating -  generateRating(5);
+// food_rating -  generateRating(5);
+// service_rating -  generateRating(5);
+// ambience_rating -  generateRating(5);
+// value_rating -  generateRating(5);
```

Comment:
is this code useful / necessary to keep?

Reply:
This was just some scaffolding to be refactored